jstests/ssl/x509/certs.yml

```diff
@@ -558,7 +558,7 @@ certs:
   - 'trusted-ca.pem'
```

Comment:
I think the other parts of this file need to be tagged as well, like the other ECDSA certs and the other OCSP certs.